photonlib-java-examples/bin/main/org/photonlib/examples/examples.json

```diff
@@ -0,0 +1,50 @@
+[
```

Comment:
I never confirmed - is the bin folder reproduced at build time? Or needs committed?

Reply:
bin should not be comitted